pkgs/applications/networking/instant-messengers/qq/update.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p curl nix-prefetch-git jq
+
+set -euo pipefail
+
```

Comment:
```suggestion

cd $(readlink -e $(dirname "${BASH_SOURCE[0]}"))

```

When running `nix-shell maintainers/scripts/update.nix --argstr package qq` the work dir is the root dir.

Reply:
done, thanks!